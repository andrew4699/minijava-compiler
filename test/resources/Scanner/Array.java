int x = new int[5];
x[0] = 1;
System.out.println(x.length);
