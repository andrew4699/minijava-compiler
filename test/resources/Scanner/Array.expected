INT ID(x) BECOMES NEW INT LBRACKET INT(5) RBRACKET SEMICOLON
ID(x) LBRACKET INT(0) RBRACKET BECOMES INT(1) SEMICOLON
DISPLAY LPAREN ID(x) DOT LENGTH RPAREN SEMICOLON
